lib/tether/toolbox.go

```diff
@@ -293,7 +293,9 @@ func toolboxOverrideArchiveRead(u *url.URL, tr *tar.Reader) error {
 
 		// no need to join on u.Path here. u.Path == spec.Rebase, but
 		// Unpack will rebase tar headers for us. :thumbsup:
-		err = archive.InvokeUnpack(op, tr, spec, diskPath)
+		op.Debugf("Unpacking tar archive to %s", diskPath)
+		// err = archive.InvokeUnpack(op, tr, spec, diskPath)
+		err = archive.Unpack(op, tr, spec, diskPath, "/.tether/unpack")
```

Comment:
This should be in `cmd/tether` - the `lib/tether` code is built into `vic-init` as well.
This _might_ require a re-org of toolbox code in general - we never really defined how guest copy operations are expected to work on the endpointVM.

The only use-case I can think of for it currently is collecting the logs if vicadmin cannot be used for some reason. I cannot think of a solid use-case for putting files onto the endpointVM.

Reply:
I can move this file over to `cmd`. I see the issue that you're pointing out.

Reply:
OK, looking into this, there are a lot of tangled dependencies, and maybe the refactor you're requesting (which is to move `toolboxOverrideArchiveRead` to `cmd`, I believe) will fully obfuscate this PR, because we at least need to move all of the container-related functions.. maybe open an issue so I can assign it to myself and handle that in a different PR @hickeng ?

Reply:
ok